subsys/bluetooth/controller/ll_sw/lll_chan.c

```diff
@@ -129,15 +129,19 @@ void lll_chan_sel_2_ut(void)
 }
 #endif /* RADIO_UNIT_TEST */
 
-/* Attribution:
- * http://graphics.stanford.edu/%7Eseander/bithacks.html#ReverseByteWith32Bits
- */
 static u8_t chan_rev_8(u8_t b)
```

Comment:
Would it make sense to flag implementation with #if defined(CONFIG_BT_CTLR_LOW_LAT) #else ?

Reply:
Ok, I will profile the bithack on CM0 and CM4. Using the loop allows the compiler to unroll it, I guess that is why its faster on CM0.